frontend/src/components/Navigation/MobileNavbar.tsx

```diff
@@ -61,7 +61,7 @@ export const MobileNavbar = () => {
                     className="mobile-navbar__link"
                     onClick={() => {
                       setIsModalOpen(true);
-                      setMenuOpen(false);
+                      setMenuOpen(true);
```

Comment:
Shouldn't this be false? When you click on a link it'll close the mobile nav menu right?